### DISCUSSION TITLE
V4 TextInput

### DISCUSSION THREAD

**Discussion by WildEgo** [nativewind/nativewind#533] [Q&A]
Hey guys, was messing around with V4 in kinda of a preview way since I saw the [prep for merge to main](https://github.com/marklawlor/nativewind/commit/6e50b035e681a72385b4c86c2c8f26ca6710a763) commit, and I noticed that the TextInput component does not follow the `className` prop, was wondering if it's just me or it just wasn't implemented yet

**Comment by atanaskanchev** [comment]
I've also tried v4, but it wasn't applying styling properly (including text input) in my expo-router based monorepo. I suppose we'll have to wait for the official release

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
